grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/redisqueue/RedisInflightTaskConsumer.java

```diff
@@ -42,10 +43,11 @@
  * @author Domenico Corapi
  */
 public class RedisInflightTaskConsumer extends TimerTask {
-    private final static Logger LOG = LoggerFactory.getLogger(RedisInflightTaskConsumer.class);
-    private final static ObjectMapper objectMapper = new ObjectMapper();
+    private static final Logger LOG = LoggerFactory.getLogger(RedisInflightTaskConsumer.class);
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    static final int ITERATIONS = 10;
```

Comment:
Not 100% sure, it's quite low level. I'll leave it there for now but if I end up changing it once I'll move it to conf